plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableQueryTests.php

```diff
@@ -597,4 +597,16 @@ public function test_query_s_filters_customer_email() {
 		$query = new OrdersTableQuery( $query_args );
 		$this->assertEqualsCanonicalizing( array( $orders[0] ), $query->orders );
 	}
+
+	/**
+	 * @testDox The get_args method returns the initial args sent.
+	 */
+	public function test_get_args() {
```

Comment:
```suggestion
	public function test_get_query_args() {
```

Reply:
Ah, good catch, thanks! Changed in 50cd2a9